cbirrt2/cbirrtparameters.h

```diff
@@ -90,12 +90,17 @@ class CBirrtParameters : public PlannerBase::PlannerParameters
         if( !PlannerParameters::serialize(O) )
             return false;
 
+        std::streamsize old_precision = O.precision();
+        O.precision(17);

```

Comment:
The way you're saving/restoring state is fine, but I thought [this was an interesting post](http://stackoverflow.com/questions/4217704/roll-back-or-undo-any-manipulators-applied-to-a-stream-without-knowing-what-th) on SO about how to save and restore stream state en bulk. Just thought I'd share. :smile: